core/templates/pages/exploration-editor-page/services/exploration-rights-backend-api.service.spec.ts

```diff
@@ -0,0 +1,222 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for Exploration Rights Backend Api Service
+ */
+
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { fakeAsync, flushMicrotasks, TestBed, waitForAsync} from '@angular/core/testing';
+import { ExplorationRightsBackendApiService } from './exploration-rights-backend-api.service';
+
+describe('Exploration Rights Backend Api Service', () => {
+  let service: ExplorationRightsBackendApiService;
+  let httpTestingController: HttpTestingController;
+  let successHandler = jasmine.createSpy('success');
+  let failHandler = jasmine.createSpy('fail');
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [ExplorationRightsBackendApiService]
+    });
+    httpTestingController = TestBed.inject(HttpTestingController);
+    service = TestBed.inject(ExplorationRightsBackendApiService);
+  }));
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('should put requestData when makeCommunityOwnedPutData called', fakeAsync(
```

Comment:
the test description can be improved. You may go through the FE Testing Wiki & read other similar test descriptions

Reply:
tried to make the test name more clear